infrastructure/ansible/roles/ats/defaults/main.yml

```diff
@@ -23,15 +32,25 @@ ort_traffic_ops_url:
 ort_traffic_ops_username:
 ort_traffic_ops_password:
 
-# ORT cronjobs to deploy
+# t3c cronjobs to deploy
 ort_crontab:
   # Perform configuration polling that doesn't require a restart on a 20 minute +- dispersion interval
   syncds:
     schedule: '0,20,40 * * * *'
     user: root
-    job:  "traffic_ops_ort.pl syncds warn {{ ort_traffic_ops_url }} '{{ ort_traffic_ops_username }}:{{ ort_traffic_ops_password }}' --login_dispersion=35 --dispersion=420 &>/tmp/ort/syncds.log"
+    job: >
```

Comment:
See above whitespace and version quoting

Reply:
Fixed